Bottom: 6d83d863086e4c1629d13b9684b8898be73b41c1
Top:    7e44f1cc4513a6c3de1582ba1fbc7aae0063987d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-27 05:07:29 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 97f92a2..21ba540 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2462,7 +2462,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
